### PR TITLE
feat(mojaloop/#3380): include SDK enhancements for R2P in Helm v15.0.1

### PR DESCRIPTION
**Pull request by mdebarros**
feat(mojaloop/#3380): include SDK enhancements for R2P in Helm v15.0.1 - https://github.com/mojaloop/project/issues/3380
- see release notes for v15.1.0

### CODE REVIEW

File: .changelog/release-v15.0.1.md [hunk 1]
```diff
@@ -10,19 +10,41 @@ Date | Revision | Description
 
 ## 0. Summary
 
-Minor parch release for [v15.0.0 Release](./release-v15.0.0.md) includes:
+Minor parch release for [v15.0.0 Release](./release-v15.0.0.md), which includes the following changes:
```
Comment (elnyry-sam-k) [inline .changelog/release-v15.0.1.md:13]:
```suggestion
Patch release for [v15.0.0 Release](./release-v15.0.0.md), which includes the following changes:
```

Comment (vijayg10) [inline .changelog/release-v15.0.1.md:13]: `parch` -> `patch`

**Comment by mdebarros** [comment]
Screenshot 2023-06-20 at 18 18 21

```
--------------------FINAL REPORT--------------------

Test Suite:GP Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 2556                          │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 2556                          │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 610                           │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 131                           │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:48:31 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:51:35.232Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 184231 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-gp_tests-2023-06-20T15:51:35.232Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:Bulk Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 337                           │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 337                           │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 63                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 16                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:51:44 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:53:46.276Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 121823 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-bulk_tests-2023-06-20T15:53:46.276Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:Thirdparty Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 28                            │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 28                            │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 20                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 8                             │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:54:28 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:54:33.850Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 4983 ms                       │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-thirdparty_tests-2023-06-20T15:54:33.850Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:SDK Bulk Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 217                           │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 217                           │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 45                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 22                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:55:16 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:57:49.648Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 152856 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-sdk_bulk_tests-2023-06-20T15:57:49.648Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:SDK Request To Pay Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 27                            │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 27                            │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 2                             │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 1                             │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:57:58 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:57:59.635Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 1183 ms                       │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-sdk_r2p_tests-2023-06-20T15:57:59.635Z.html was generated
```

**Comment by mdebarros** [comment]
## Helm 

```
❯ helm list
NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION                                                                                                                                                                                                                                                                                                                                                                                                                   
backend moja3           1               2023-06-20 17:28:36.682134 +0200 SAST   deployed        example-mojaloop-backend-15.0.0 nginx: 4.4.2 ; mysql: 9.4.1; kafka: 19.0.2; mongodb: 13.3.1; redis:17.3.7                                                                                                                                                                                                                                                                                                                                                     
moja3   moja3           1               2023-06-20 17:33:24.805551 +0200 SAST   deployed        mojaloop-15.1.0                 ml-api-adapter: v14.0.0; central-ledger: v17.0.3; account-lookup-service: v13.0.0; quoting-service: v15.0.2; central-settlement: v15.0.0; bulk-api-adapter: v15.0.3; transaction-requests-service: v14.0.1; simulator: v12.0.0; mojaloop-simulator: v14.0.1; sdk-scheme-adapter: v23.0.1; auth-service: v14.0.0; consent-oracle: v0.2.0; thirdparty-sdk: v13.0.2; ml-testing-toolkit: v16.0.1; ml-testing-toolkit-ui: v15.2.1;
```

See the following for config changes used for testing: [staged.IGNORE.patch](https://github.com/mojaloop/helm/files/11803372/staged.IGNORE.patch)


## Test Results

![Screenshot 2023-06-20 at 18 18 21](https://github.com/mojaloop/helm/assets/11392873/4624b2bc-297c-447b-8fde-8d6cbd9fe0b9)

```
--------------------FINAL REPORT--------------------

Test Suite:GP Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 2556                          │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 2556                          │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 610                           │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 131                           │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:48:31 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:51:35.232Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 184231 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-gp_tests-2023-06-20T15:51:35.232Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:Bulk Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 337                           │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 337                           │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 63                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 16                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:51:44 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:53:46.276Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 121823 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-bulk_tests-2023-06-20T15:53:46.276Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:Thirdparty Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 28                            │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 28                            │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 20                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 8                             │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:54:28 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:54:33.850Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 4983 ms                       │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-thirdparty_tests-2023-06-20T15:54:33.850Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:SDK Bulk Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 217                           │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 217                           │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 45                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 22                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:55:16 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:57:49.648Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 152856 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-sdk_bulk_tests-2023-06-20T15:57:49.648Z.html was generated
```

```
--------------------FINAL REPORT--------------------

Test Suite:SDK Request To Pay Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 27                            │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 27                            │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 2                             │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 1                             │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 20 Jun 2023 15:57:58 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ 2023-06-20T15:57:59.635Z      │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 1183 ms                       │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-sdk_r2p_tests-2023-06-20T15:57:59.635Z.html was generated
```

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**